docs/guides/Quick Start/03-orders.md

```diff
@@ -0,0 +1,182 @@
+# Order Overview
+
+The order resource enables merchants to process orders they receive from customers.
+
+You can use the orders resource to do the following:
+
+* Retrieve orders and their line-items
+* Create dummy orders 
+* Update order statuses
+* Manage returns
+
+## Retrieve Orders
+
+A list of all customer orders is obtained using the route:
+
+``` markdown
+POST /api/search/order
+```
+
+### Order line-items
+
+An order can have more other items or child items of `type` : `product`, `promotion`, `credit` or `custom`. To fetch line items for a particular order, try the below route:
+
+```text
+GET /api/order/{order-id}/line-item
+```
+Sample API request:
+
+```json http
+{
+  "method": "get",
+  "url": "http://localhost/api/order/558efc15fe604829b4d0607df75187e0/line-item",
+  "headers": {
+    "Content-Type": "application/json",
+    "Accept": "application/json",
+    "Authorization": "Bearer Your_API_Key"
+  }
+  }
+```
+
+## Create an order
+
+You can create orders manually in the admin panel to record orders made outside of Shopware or send customer invoices.
```

Comment:
In here maybe explain the limitation of creating orders that way - e.g. no payment attached, no invoices are created automatically etc.